src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  --normal-color: black;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background-color: white;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.nav__header {
+  display: flex;
+  align-self: flex-end;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  margin-left: 20px;
```

Comment:
crane elements should not have indents
try using this:

```
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```

![opera_pKJ1r8Z0l8](https://github.com/mate-academy/layout_moyo-header/assets/104684446/5e2fe7ec-24bf-4c98-862e-5770a30b371f)